### DISCUSSION TITLE
Any plans for new release?

### DISCUSSION THREAD

**Discussion by marknefedov** [alicevision/Meshroom#1835] [Q&A] [answered]
Hello, are there any plans for new release? The latest release was in early 2021, and it looks like Meshroom acquired a significant number of changes since then.

**Comment by natowi** [comment, marked as answer]
There is still some work to be done for a release:
https://github.com/alicevision/Meshroom/milestone/13
https://github.com/alicevision/AliceVision/milestone/33
